### DISCUSSION TITLE
Github Copilot X (Chat) dealing with sensitive data

### DISCUSSION THREAD

**Discussion by zhyph** [community/community#56367] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello,

Just got access to Copilot X, and I was wondering, atm, how reliable is it around sensitive information? Is it the same as the regular Copilot? Maybe there was some type of README or even while accepting the terms, that I may have missed out, in the FAQ on the waitlist page there's nothing mentioning it, so I was just wondering.

Thanks.

**Comment by davecheney** [comment, marked as answer]
@zhyph the terms and conditions for Copilot Chat is the same as Copilot for code completions. https://github.com/settings/copilot will let you choose if you wish to permit us to record your prompts and completion telemetry.

**Reply by zhyph** [reply]
Good to know, thanks @davecheney

**Reply by rajdotio** [reply]
@davecheney I don't see any such option under my org's Copilot settings. What does that mean?
